pkg/apis/pipeline/pod/template.go

```diff
@@ -101,6 +101,11 @@ type Template struct {
 	// +optional
 	ImagePullSecrets []corev1.LocalObjectReference `json:"imagePullSecrets,omitempty"`
 
+	// HostAliases is an optional list of hosts and IPs that will be injected into the pod's hosts
+	// file if specified. This is only valid for non-hostNetwork pods.
+	// +optional
+	HostAliases []corev1.HostAlias `json:"hostAliases,omitempty" protobuf:"bytes,23,rep,name=hostAliases"`
```

Comment:
NIT: I don't think `protobuf` has any significance here and not needed, but we have it included in many other fields in this `struct`

Reply:
Should i remove that?

Reply:
yup sure, thanks @xclud

Not related to change here, but the `struct` looks odd with some fields having `json` and `protobuf` both and some with just `json` 😜  We have been using `json` tags throughout the entire code base.